src/originalBudgetTrackingApp/export/outputVendors/csv/csv.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  EnrichedTransaction, ExportTransactionsFunction, OutputVendor, OutputVendorName
+} from '@/originalBudgetTrackingApp/commonTypes';
+import { mergeTransactions, sortByDate } from '@/originalBudgetTrackingApp/transactions/transactions';
+import { TransactionInstallments } from '@brafdlog/israeli-bank-scrapers-core/lib/transactions';
+import { promises as fs } from 'fs';
+
+const stringify = require('csv-stringify/lib/sync');
+const parse = require('csv-parse/lib/sync');
+
+export function parseTransactions(csvText: string) {
+  return parse(csvText, {
+    columns: true,
+    cast: (value, context) => {
+      switch (context.column) {
+        // note: we check here some number-type values.
+        // if there are undefined string-type values (memo?, for example) they'll be serialized as empty string.
+        // it means that the deserialized object will not be 100% identical to the serialzed.
+        // I'm not sure how to represent empty string differently than undefined string.
+        case 'chargedAmount':
+        case 'identifier':
+        case 'originalAmount':
+          if (value === '') {
+            return undefined;
+          }
+          return parseFloat(value);
+        case 'installments':
+          if (value === '') {
+            return undefined;
+          }
+          {
+            const separator = value.indexOf('/');
+            if (separator !== -1) {
+              const installments: TransactionInstallments = {
+                number: parseInt(value.substring(0, separator - 1), 10),
+                total: parseInt(value.substring(separator + 1), 10)
+              };
+              return installments;
+            }
+            return value;
+          }
+        default:
+          return value;
+      }
+    }
+  }) as EnrichedTransaction[];
+}
+
+export const serializeTransactions = (transactions: EnrichedTransaction[]) => {
```

Comment:
`export` is to make this method accessible outside the file, by assigning it as a property of the module. For example:

```nodejs
import myModule from 'myFile'
myModule.serializeTransactions()
```

But you are not using this function outside the file.

Reply:
I'm using it in the test file test/unit/specs/csv.spec.ts
Is there a way of testing it without exposing it with export?

Reply:
@brafdlog Baruch is in Miluim, maybe you have an idea?

Reply:
You can leave the export if you do test the function.

In general, we could use something like rewire for this but I don't see a need for that at this stage.

Reply:
ok